src/mlpack/methods/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ set(DIRS
   local_coordinate_coding
   logistic_regression
   lsh
+  lsh_model
```

Comment:
Do you think we should just put the LSH modeling code in the `lsh` directory?  I don't see a problem with keeping them together.